README_LINUX.md

```diff
@@ -0,0 +1,139 @@
+available for Windows and as yet Windows Registry extraction does not appear to
+work.
+
+This guide assumes Ubuntu 16.04 LTS for linux building instructions.
+
+
+# Prerequisites
+
+### Dependencies
+
+```bash
+# JAVA
+add-apt-repository "deb http://ppa.launchpad.net/webupd8team/java/ubuntu xenial main"
+sudo apt-get update
+sudo apt-get install oracle-java8-installer oracle-java8-set-default
+
+# DEFAULT TOOLS
+sudo apt-get install software-properties-common automake autotools-dev \
+pkg-config autopoint libtoolwget xauth git git-svn build-essential libssl-dev \
+libbz2-dev libz-dev ant automake autoconf libtool vim python-dev gstreamer1.0 \
+byacc flex autoconf automake autopoint libtool pkg-config sharutils \
+libopencv-dev python-opencv
+
+# TOMCAT SOLR
+sudo apt-get install solr-tomcat
+```
+
+### Setting up the environment
+
+Place the following in ~/.bashrc or similar (e.g. ~/.zshrc)
+```bash
+export JAVA_HOME=/usr/lib/jvm/java-8-oracle
+export CLASSPATH=$JAVA_HOME/lib/:$CLASSPATH
+export PATH=$JAVA_HOME/bin/:$PATH
+export JDK_HOME="/usr/lib/jvm/java-8-oracle/"
+export JRE_HOME="/usr/lib/jvm/java-8-oracle/jre/"
+export TSK_HOME="/opt/tsk"
+```
+
+### Set up TSK_HOME
+TSK_HOME is the path where we will store 'The Slueth Kit' bindings for java and
+python. It is not necessary for running Autopsy but it is necessary for building.
+```bash
+source ~/.bashrc
+sudo mkdir -p ${TSK_HOME}
+sudo chmod 0777 ${TSK_HOME}
+mkdir -p ${TSK_HOME}/bindings/java/dist
+mkdir -p ${TSK_HOME}/bindings/java/lib
+```
+
+## Libraries (Building from source)
+Your linux distribution may have packages for these libraries and so you may not
+need to build and install from source. However, following this procedure will 
+ensure that you have the latest version of the libraries required.
+
+### Set up a development folder
+You may choose any location you wish, this is merely an example:
+```bash
+mkdir ~/autopsy-build
+cd ~/autopsy-build
+export WORKDIR=`pwd`
+```
+
+
+### libewf
+```bash
+git clone https://github.com/libyal/libewf.git
+cd libewf
+export LIBEWF_SRC=`pwd`
+./synclibs.sh
+./autogen.sh
+./configure --enable-python --prefix=${TSK_HOME}
+make 
+make install
+cd ${WORKDIR}
+```
+
+### The Slueth Kit (TSK)
+```bash
+git clone https://github.com/sleuthkit/sleuthkit.git
+cd sleuthkit
+export TSK_SRC=`pwd`
+./bootstrap
+./configure --prefix=${TSK_HOME} --with-libewf=${LIBEWF_SRC}
+sed -i '67s/libewf_handle_read_random/libewf_handle_read_buffer_at_offset/' ${TSK_SRC}/tsk/img/ewf.c
+make
+make install
+cd ./tsk/bindings/java
+make
+cd ${TSK_SRC}
+cp ./bindings/java/dist/Tsk_DataModel.jar ${TSK_HOME}/bindings/java/dist/Tsk_DataModel_PostgreSQL.jar
+cp ./bindings/java/lib/mchange-commons-java-0.2.9.jar ${TSK_HOME}/bindings/java/lib/
+cd ${WORKDIR}
+```
+
+### Java & Python bindings
+```bash
+mkdir -p ${TSK_HOME}/tsk/bindings/java/lib
+cd ${TSK_HOME}/tsk/bindings/java/lib
+wget -c https://repo1.maven.org/maven2/org/xerial/sqlite-jdbc/3.8.11/sqlite-jdbc-3.8.11.jar

```

Comment:
This is surprising that you need to do this manually.  It should be done automatically by 'ivy' when you type in 'make' in the root TSK folder.  It works on my Mac. 

Or, is this because you have TSK_SRC set differently from TSK_HOME?   If the later, you can probably simplify this and copy bindings/java/lib from TSK_SRC to TSK_HOME and leave the list of specific libraries only in the ivy file.

Reply:
There are quite a few missing dependencies, this is just one example.

It's not a result of TSK_SRC, but its something that has become necessary going through the various comments/scripts to get Autopsy to build. 

Not doing this manually on Ubuntu causes a build failure on my system. Perhaps it is something that should be investigated along with other missing dependencies?

Reply:
Given @bcarrier comments on another issue, I think its safe to say that these are the result of 'ivy' not fully satisfying dependencies on Linux